include/dpp/cluster.h

```diff
@@ -747,6 +747,21 @@ class cluster {
 	 */
 	void on_integration_delete (std::function<void(const integration_delete_t& _event)> _integration_delete);
 
+	/**
```

Comment:
would you like for me to add comments for these events after its merged, or are you wanting to do this now? We'd need comments either way :)